README.md

```diff
@@ -82,6 +85,39 @@ inside your app project to install the module.
 </resources>
 ```
 
+##### AndroidManifest.xml
```

Comment:
I believe this is a required necessity for {N} 4.0?

Reply:
Yes, in 4.0 the NativeScript CLI will try to package the plugin manifest in an .aar. When the plugin manifest contains resources that are not in the plugin, the process will fail. In the case of the social login plugin, the manifest contains string resources for the Facebook configuration but the strings are supposed to be defined in the application resources. The CLI will try to package the plugin and will fail to resolve the Facebook config strings. Moving the configuration out of the plugin manifest and into the application's manifest (where the strings are) will fix this.

Reply:
Oh! Thanks. Merging then. 👍